source/DistanceAndDirection/ProAppDistanceAndDirectionModule/ProAppDistanceAndDirectionModule/ViewModels/ProLinesViewModel.cs

```diff
@@ -168,7 +168,18 @@ private void UpdateManualFeedback()
                 }).Result;
 
                 if (segment != null)
-                    UpdateFeedbackWithGeoLine(segment);
+                    await UpdateFeedbackWithGeoLine(segment);
+            }
+            else if (LineFromType == LineFromTypes.Points && HasPoint1 && Point1 != null && HasPoint2 && Point2 != null)
+            {
+                // update feedback
+                var segment = QueuedTask.Run(() =>
+                {
+                    return LineBuilder.CreateLineSegment(Point1, Point2);
+                }).Result;
+
+                if (segment != null)
+                    await UpdateFeedbackWithGeoLine(segment);
```

Comment:
this will need to be removed also, I had to update this in a different way to apply to all tabs